src/css-grids/cssgrids-style.css

```diff
@@ -0,0 +1,427 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-image: url(https://images.unsplash.com/photo-1534543210152-32025bcfaad9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: top;
+}
+
+h1, h2, h3, h4, h5, h6, p, a, span, ul, .calendar {
+    font-family: 'Muli', sans-serif;
+}
+
+
+main.main-container {
+    display: grid;
+    grid-template-columns: 1fr 300px 1fr 450px 1fr 280px 1fr;
+    grid-template-rows: 50px repeat(11, 75px);
```

Comment:
when you go beyond the `max-width:1099px`, 11 rows creates a scroll bar with no info or elements below these three. maybe start with `repeat(9, 75px)` as default.